R/tt_pos_and_access.R

```diff
@@ -280,6 +280,34 @@ setMethod("tt_at_path", "VTableTree",
 
 #' @export
 #' @rdname ttap
+#' 
+#' @examples 
```

Comment:
Please add text specifying that assigning `NULL` removes the subtable at the path, in addition to spelling that out in the example?